packages/apollo-voyager-audit/src/index.ts

```diff
@@ -0,0 +1,75 @@
+import { buildPath } from '@aerogear/apollo-voyager-tools'
+import { GraphQLResolveInfo } from 'graphql'
+import { IFieldResolver } from 'graphql-tools'
+import pino from 'pino'
+
+const log = pino()
+const auditLogger = log.child({tag: 'AUDIT'})
+
+let auditLogEnabled = false
+
+export function isAuditLogEnabled (): boolean {
+  return auditLogEnabled
+}
+
+export function setAuditLogEnabled (val: boolean): void {
+  auditLogEnabled = val
+}
+
+interface ResolverObject {
+  [key: string]: IFieldResolver<any, any>
+}
+
+export function wrapResolversForAuditLogging (resolverMappings: { [key: string]: ResolverObject }): { [key: string]: ResolverObject } {
+  const output: { [key: string]: ResolverObject } = {}
+
+  const typeKeys = Object.keys(resolverMappings)
+  for (const typeKey of typeKeys) {
```

Comment:
Seeing as we have this exact thing in two places, it might make sense to move it into `apollo-voyager-tools` at some stage. It could accept the wrapper function as an argument and look after the rest